src/test/java/com/google/api/codegen/testdata/discoveries/go/go_compute.v1.json.baseline

```diff
@@ -4041,10 +4041,10 @@ func main() {
     // TODO: Handle error.
   }
 
-  // Project ID for this request.
+  // * Project ID for this request.
   project := "" // TODO: Update placeholder value.
 
-  // The name of the zone where the instance group is located.
+  // * The name of the zone where the instance group is located.
```

Comment:
This only looks strange in Go because of the other inconsistency you mentioned, which will be fixed. When there is a general TODO above all the placeholders, it appears logical to offset the comments for each one so that the extent of the placeholders section is clear, as was previously done in the Python samples.